recipe-server/client/control/state/actions/actions.js

```diff
@@ -7,8 +7,12 @@ import {
 } from '../requests/actions';
 
 
-function receiveAction(action) {
-  return dispatch => {
+export function fetchAction(name) {
+  return async dispatch => {
+    const requestId = `fetch-action-${name}`;
+    const response = dispatch(makeApiRequest(requestId, `action/${name}/`, { method: 'GET' }));
+    const action = await response;
```

Comment:
oh i guess there's no real benefit i was just trying to keep the line wrap down and this looked a little less cluttered to me. but i can ditch the extra line if you'd prefer to see it that way. i'm not tied to this.